### DISCUSSION TITLE
Request for help regarding electrode location outside the head

### DISCUSSION THREAD

**Discussion by shntrk** [simnibs/simnibs#292] [2. General]
Hello, in my research I used 2 mA 20 minutes anodal tDCS. Anode electrode placed in Fpz while cathode electrode placed in left shoulder. I want to create a tDCS stimulation modelling but I didn't understand that how I should placed cathode electrode on head models (there is not appeared shoulders) in simNIBS. If you help me I will be very thankful. Thank you in advance.

I wish good work,

**Comment by oulap** [comment]
We do not currently have a head model with the shoulders included, but as an approximation you can place the electrode on the neck of the head model.
